admin/app/templates/views/terms-of-use.html

```diff
@@ -13,46 +13,393 @@
-
+    <p>
+      If you <strong>don’t</strong> keep to these terms, we may cancel your
+      subscription and your messages may not be sent.
+    </p>
     <p>Notify will:</p>
     <ul class="list list-bullet">
-      <li>send all the messages you pass to us, as long as they meet our guidelines</li>
-      <li>show how Notify is performing through our <a href="https://status.notify.gov.au/">status</a> page.</li>
-      <li>keep your data <a href="{{ url_for('.security') }}">secure</a></li>
-      <li>give you one month’s notice by email if we change our terms of use or delivery providers</li>
+      <li>
+        send all the messages provided that they meet the terms of use
+      </li>
+      <li>
+        show how Notify is performing through our
+        <a href="https://status.notify.gov.au/">status</a>
+        page
+      </li>
+      <li>keep your data secure</li>
+      <li>
+        give one month’s notice, by email, if we change our Terms of Use or
+        delivery providers
+      </li>
     </ul>
-
-    <h2 class="heading-medium">Leaving Notify</h2>
-    <p>You can leave Notify at any time. Just <a href="{{url_for('.feedback', ticket_type='ask-question-give-feedback')}}">contact us</a> and we’ll close your account.</p>
-    <p>When you leave Notify, all your data will be deleted.</p>
-
+    <p>
+      The DTA may change these Terms of Use from time to time and will advise
+      users of any changes by email or telephone.
+    </p>
+    <p>
+      If you have any questions or feedback about these Terms of Use, please
+      <a href="mailto:notify-support@digital.gov.au">contact us</a>.
+    </p>
+    <h2 class="heading-medium" id="subscribe">Subscribe</h2>
+    <p>
+      The Notify service is open to Commonwealth, State, Territory and Local
+      government bodies.
+    </p>
+    <h3><strong>Trial mode</strong></h3>
+    <p>
+      <a href="mailto:notify-support@digital.gov.au">Contact us</a> to set up a
+      free trial account for your service and begin using it immediately. In
+      trial mode:
+    </p>
+    <ul class="list list-bullet">
+      <li>
+        you can only send messages to yourself
+      </li>
+      <li>
+        you can add people to your team, then you can send messages to them too
+      </li>
+      <li>
+        you can only send 50 messages per day
+      </li>
+    </ul>
+    <h3><strong>Go live</strong></h3>
+    <p>
+      You can request to go live, when you have:
+    </p>
+    <ol class="list list-number">
+      <li>
+        Accepted our terms of use
+      </li>
+      <li>
+        Have more than one team member with the ‘Manage service’ permission
+      </li>
+      <li>
+        Create templates showing the kind of messages you plan to send
+      </li>
+    </ol>
+    <h2 class="heading-medium" id="features">Features</h2>
+    <p>With Notify, you can:</p>
+    <ul class="list list-bullet">
+      <li>Send emails</li>
```

Comment:
Previous bullet points start with lower case. These ones and the ordered lists start with capitals.